content/docs/user-guide/components/development/expose-animation.md

```diff
@@ -1,8 +1,9 @@
 ---
-description: ' Learn how to expose custom components and their properties to the <guilabel>Track View</guilabel> editor
-  for animation in Open 3D Engine. '
+linkTitle: Exposing Custom Components to Track View for Animation
 title: Exposing Custom Components to Track View for Animation
-weight: 900
+description: Learn how to expose custom components and their properties to the <guilabel>Track View</guilabel> editor for animation in Open 3D Engine. 
```

Comment:
```suggestion
 description: Learn how to expose custom components and their properties to the Track View editor for animation in Open 3D Engine. 
```

Reply:
Thanks for catching that, I hadn't seen that token elsewhere in the project.